src/CssInliner.php

```diff
@@ -421,9 +421,11 @@ static function (array $m): string {
         // in order to not overwrite existing style attributes in the HTML, we
         // have to save the original HTML styles
         $nodePath = $node->getNodePath();
-        if (!isset($this->styleAttributesForNodes[$nodePath])) {
-            $this->styleAttributesForNodes[$nodePath] = $this->parseCssDeclarationsBlock($normalizedOriginalStyle);
-            $this->visitedNodes[$nodePath] = $node;
+        if (\is_string($nodePath)) {
+            if (!isset($this->styleAttributesForNodes[$nodePath])) {
+                $this->styleAttributesForNodes[$nodePath] = $this->parseCssDeclarationsBlock($normalizedOriginalStyle);
+                $this->visitedNodes[$nodePath] = $node;
+            }
         }
```

Comment:
Sorry, I just noticed this which I missed first time round.  We could use `&&` rather than two separate `if` statements.

Reply:
Done.